### ISSUE TITLE
ValueError: Either low-pass or high-pass needs to be set for band-pass

### ISSUE THREAD

**Issue by Phaips** [SBC-Utrecht/pytom-template-matching-gpu#61]
Hey,

I get an error testing the new version with the full 3D CTF and wedge model.
My inputs are:

pytom_match_template.py -t /../template_tomo_612.mrc --mask mask.mrc -v tomo/612.rec -d /612 -a temp/612.tlt -s 2 2 1 --angular-search 7.00 -g 0 1 2 3 --voxel-size-angstrom 7.84 --non-spherical-mask --per-tilt-weighting --amplitude-contrast 0.07 --dose-accumulation temp/612_exposure.txt --defocus-file temp/612_defocus.txt --voltage 300 --spherical-abberation 2.7

and the error comes from the weights.py

```
/../miniconda3/envs/pytom_tm_v0.3.1/lib/python3.9/site-packages/pytom_tm/weights.py", line 131, in create_gaussian_band_pass
    raise ValueError('Either low-pass or high-pass needs to be set for band-pass')
ValueError: Either low-pass or high-pass needs to be set for band-pass
```

Since we never lp or hp filtered our tomos and never had this issue in version 0.2 I am not sure where the error is coming from.

**Comment by McHaillet** [comment]
Thanks for the report! Will try to make a fix for that ASAP

**Comment by McHaillet** [comment]
And btw, setting the low-pass to the nyquist resolution should already allow you to run it right now without removing signal.

**Comment by Phaips** [comment]
> And btw, setting the low-pass to the nyquist resolution should already allow you to run it right now without removing signal.

Thanks for the quick reply! This is already a great solution and it works!

**Comment by McHaillet** [comment]
Issue has been solved with PR #62 !
